yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/commons/DockerCreateContainer.java

```diff
@@ -369,6 +374,24 @@ public void setEnvironmentString(String environmentString) {
         setEnvironment(splitAndFilterEmpty(environmentString));
     }
 
+    @Nonnull
+    public String getDockerLabelsString() {
+        return joinToStr(dockerLabels);
+    }
+
+    @DataBoundSetter
+    public void setDockerLabelsString(String dockerLabelsString) {
+        setDockerLabels(splitAndFilterEmpty(dockerLabelsString));
+    }
+
+    public void setDockerLabels(List<String> dockerLabels) {
+        this.dockerLabels = dockerLabels;
+    }
+
+    public List<String> getDockerLabels() {
+        return dockerLabels;
```

Comment:
add null check and return empty list, either it will through NPE after udpate because xstream for new field will set null.